include/pacemaker.h

```diff
@@ -26,6 +26,35 @@ extern "C" {
 
 #  include <crm/stonith-ng.h>
 
+/*!
+ * \brief Control which portions of cluster status are displayed
+ */
+typedef enum {
+    pcmk_show_stack         = 1 << 0,
+    pcmk_show_dc            = 1 << 1,
+    pcmk_show_times         = 1 << 2,
+    pcmk_show_counts        = 1 << 3,
+    pcmk_show_options       = 1 << 4,
+    pcmk_show_nodes         = 1 << 5,
+    pcmk_show_resources     = 1 << 6,
+    pcmk_show_attributes    = 1 << 7,
+    pcmk_show_failcounts    = 1 << 8,
+    pcmk_show_operations    = 1 << 9,
+    pcmk_show_fence_failed  = 1 << 10,
+    pcmk_show_fence_pending = 1 << 11,
+    pcmk_show_fence_worked  = 1 << 12,
+    pcmk_show_tickets       = 1 << 13,
+    pcmk_show_bans          = 1 << 14,
+    pcmk_show_failures      = 1 << 15,
+} pcmk_show_e;
+
+#define pcmk_show_fencing_all    (pcmk_show_fence_failed | pcmk_show_fence_pending | pcmk_show_fence_worked)
+#define pcmk_show_summary        (pcmk_show_stack | pcmk_show_dc | pcmk_show_times | pcmk_show_counts)
+#define pcmk_show_all            (pcmk_show_summary | pcmk_show_options | pcmk_show_nodes | \
+                                  pcmk_show_resources | pcmk_show_attributes | pcmk_show_failcounts | \
+                                  pcmk_show_operations | pcmk_show_fencing_all | pcmk_show_tickets | \
+                                  pcmk_show_bans | pcmk_show_failures )
+
```

Comment:
Put these additions in a separate "API: lib...: add ..." commit for changelog purposes

Reply:
Fixed.